### ISSUE TITLE
Extrinsics fail due to 30s timeout

### ISSUE THREAD

**Issue by isSerge** [polkadot-js/api#4742]
Hi,

Recently my app started experiencing this error after extrinsics are submitted:
```
Error: Dropped
at /dist/target.js:49:28
at listOnTimeout (node:internal/timers:559:17)
at processTimers (node:internal/timers:502:7)
```
The app submits up to 200 extrinsics per block. I suspect this started after #4661. I tried running `"@polkadot/api": "7.12.1"` and it does not have this problem. 

I understand that is not a usual use case, but it would be nice if we could have timeout value configurable. I could submit a PR for that

**Comment by jacogr** [comment]
PRs very welcome. I always think it should go to 45s as a default. (Instead of 30s)

**Comment by jacogr** [comment]
Closing, addressed in #4758
